src/Commands/EnvDeleteCommand.php

```diff
@@ -167,6 +167,10 @@ function ($item) {
         elseif (!empty($options['preserve-if-branch'])) {
             $environmentsWithoutPRs = $this->preserveEnvsWithBranches($oldestEnvironments, $multidev_delete_pattern);
         }
+        else {
+          $environmentsWithoutPRs = $oldestEnvironments;
```

Comment:
This is too destructive. delete:env should have one or the other of preserve-prs or preserve-if-branch set.

Reply:
Removed.
I think the change in BuildToolsBase is all that's needed.